src/routes/me/definitions.ts

```diff
@@ -73,3 +72,33 @@ export type MintApiEndpoint = ApiEndpointDefinition<
   MintApiCall,
   MintApiResponse
 >;
+
+export interface MigratePayload {
+  mobileNumber: string;
```

Comment:
Can we also edit it to include email address explicitly? We will be asking for that too.

Reply:
Actual transition is happening in mobile app.